sklearn/metrics/_scorer.py

```diff
@@ -219,6 +219,9 @@ def __init__(self, score_func, sign, kwargs, response_method="predict"):
         self._sign = sign
         self._kwargs = kwargs
         self._response_method = response_method
+        # XXX After removing the deprecated scorers (v1.8) remove the
+        # XXX deprecation_msg property again and remove __call__'s body again
```

Comment:
I think there is no `__call__` method involved here, can you adapt the comment.

Also I agree this is what https://github.com/scikit-learn/scikit-learn/pull/14898 did but it looks like the convention for this kind of note to our maintainers is something like this
```py
# TODO(1.8): remove in 1.8 (+potentially add any relevant comments that help the future maintainers)
```

Reply:
Ah yes I blindly copy-pasted here and forgot to go back and change it. Sorry about that!